### DISCUSSION TITLE
Mute button UI/UX adjustment

### DISCUSSION THREAD

**Discussion by Redo7** [ppy/osu#29480] [Ideas]
### Type

Cosmetic

### Bug description

Thought I'd suggest a "better" and more clean way of having the mute button implemented into the master volume area

Currently, the mute button is awkwardly squished between the volume text and the circular border indicator. This makes things quite tight in there, so I'm proposing a few changes:
- Moving the volume button between the "ends" of the sliding indicator
  - This makes use of the free real estate in that area, while freeing up the space around the text
- Adding a backing track for the slider indicator
  - Further reinforces the effect by making it more clear why the icon is there. Otherwise it could be interpreted as a "seemingly random position"
- Replacing the current volume with a "MUTED" label when the output is muted
  - Icons are small, and can be overlooked. Adding a clear change in the UI would notify the user when they click something by accident

There are also one UX change me, along with [cyperdark](https://osu.ppy.sh/users/9893708) have thought of:
- Clicking anywhere on the volume circle could also mute it's output
  - That makes the mute button more of a mute indicator, but introduces a bigger click area for easier muting
  - As of now there is nothing happening when the Master Volume circle gets clicked. And I think clicking a volume indicator to mute it just makes sense
  - The mute button could still be clickable to have a clear purpose of it's own (this no longer makes it a mute indicator, but a button again :v)

Below I attach an image showing all the proposed visual changes
![osu!_ftUClnCTuB](https://github.com/user-attachments/assets/a59576fe-da38-458b-8c63-672fda7b6b9b)

Also as a side note
The speaker in the muted icon seems to be a tiny bit smaller than the unmuted one (it does get visibly smaller when clicking it in game)
Not sure if it's intended, but I thought I'd mention it in case it wasn't
![Photoshop_paVkF5KCFH](https://github.com/user-attachments/assets/f7dd322b-edfa-4a4e-965b-193f8b3292cd)

### Version

2024.816.0 / 2024.817.0

### Logs

N/A

**Comment by peppy** [comment]
As mentioend everywhere, there's a new design for the volume thing coming so please check on that and provide any feedback on that.

The size difference is intentional ,yes.
